src/upm_stdvector.i

```diff
@@ -0,0 +1,61 @@
+%include "autobox.i"
```

Comment:
How about upm_javastdvector.i?  Thanks for the  comments - as you can tell I'm not a experienced Java developer :)